angular2-qrcode.ts

```diff
@@ -3,34 +3,43 @@ import {
     Component,
     Input,
     ElementRef,
-    OnInit
+    OnChanges,
+    SimpleChanges
 } from '@angular/core';
-import qrcode = require('qrcode-generator');
+import * as qrcode from "qrcode-generator";
```

Comment:
If I could just suggest: 
import qrcode from "qrcode-generator";

It result with your namespace:
Cannot call a namespace ('qrcode')

Anyway thx I could make it work with angular2 with your work :)